.github/workflows/release.yml

```
@@ -0,0 +1,76 @@
+
+      - name: Build
+        run: cargo build --release --target ${{ matrix.target }}
+
+      - name: Package
+        shell: bash
+        run: |
+          cd target/${{ matrix.target }}/release
+          tar czf ../../../${{ matrix.artifact_name }} raphook*
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)